src/AspNet.Security.OAuth.DigitalOcean/DigitalOceanAuthenticationConstants.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.DigitalOcean;
+
+/// <summary>
+/// Contains constants specific to the <see cref="DigitalOceanAuthenticationHandler"/>.
+/// </summary>
+public static class DigitalOceanAuthenticationConstants
+{
+    public static class Claims
+    {
+        /// <summary>
+        /// The claim for determining if the user's email address has been verified.
+        /// </summary>
+        public const string EmailVerified = "urn:digitalocean:email_verified";
+    }
+
+    /// <summary>
+    /// DigitalOcean API Scopes
+    /// <para>https://docs.digitalocean.com/reference/api/oauth-api/</para>
+    /// </summary>
+    public static class Scopes
+    {
+        /// <summary>
+        /// Read Access to the DigitalOcean account.
+        /// <para>Default scope</para>
+        /// </summary>
+        public const string Read = "read";
+
+        /// <summary>
+        /// Write Access to the DigitalOcean account.
+        /// </summary>
+        public const string Write = "write";
+    }
+
+    /// <summary>
+    /// DigitalOcean API Response Types
+    /// <para>https://docs.digitalocean.com/reference/api/oauth-api/</para>
+    /// </summary>
+    public static class ResponseTypes
+    {
+        /// <summary>
+        /// Web Application Flow.
+        /// <para>Default scope</para>
+        /// </summary>
+        public const string Code = "code";
+    }
```

Comment:
What's the motivation for adding these constants?

They don't seem to be used in the provider, so I'd be a bit reticent to add them in if they're not needed, particularly if there's plans in the works to change how DigitalOcean's OAuth implementation works.

Reply:
I was just following the pattern and to also give some guidance to consumers as the docs come up a bit short.

You're right-- I forgot to consume them ☕

Reply:
I appreciate your thoroughness in this PR. I based my implementation on the QQ provider, which doesn't follow the same standards.

I'm removing the `ResponseTypes`.